twitter_api_tutorial/README.md

```diff
@@ -0,0 +1,49 @@
+# Getting started with the Twitter API v2 using Python
+
+This folder contains the code behind the [medium article](https://medium.com/data-analytics-at-nesta) on collecting Twitter data using Python and the Twitter API v2.
+
+To be able to run the code, you need a [developer account](https://twitter.com/i/flow/login?input_flow_data=%7B%22requested_variant%22%3A%22eyJyZWRpcmVjdF9hZnRlcl9sb2dpbiI6Imh0dHBzOi8vZGV2ZWxvcGVyLnR3aXR0ZXIuY29tL2VuL3BvcnRhbC9wZXRpdGlvbi9lc3NlbnRpYWwvYmFzaWMtaW5mbyJ9%22%7D) (the process is fairly straightforward and is also described in the [medium article](https://medium.com/data-analytics-at-nesta)).
+
+## Tutorial
+
+1. **First part: [Your first Twitter API request](https://github.com/nestauk/dap_medium_articles/blob/3_twitter_api_tutorial/twitter_api_tutorial/Your%20first%20Twitter%20API%20request.ipynb)**
+
+Learn how to make your first Twitter API request and what the collected data looks like.
```

Comment:
I know this is in the article as well - but I wonder whether at some point in this README it might be nice to include a very basic look at the final outputs? Just for people who want to see at a glance what sort of data they might be able to get out of this codebase.

e.g.
Your Twitter API request can output data like:
```
{'edit_history_tweet_ids': ['1592212117007892480'],
 'author_id': '209980898',
 'created_at': '2022-11-14T17:45:25.000Z',
 'text': "I hadn't quite appreciated just how much more common air-to-air heat pumps are than air-to-water in Europe.\nAside from Germany, Poland, Czechia and Austria, most other countries are mainly air-to-air...\nhttps://t.co/ELrBrK2yvL [https://t.co/n5cTxwLxk3",](https://t.co/n5cTxwLxk3%22,)
 'id': '1592212117007892480'}

Reply:
That sounds good, thank you @lizgzil! Would you add something like what I have in the medium post (with the details deleted) or maybe an output with mock data?

Reply:
maybe an output with mock data would be clearer